src/assets/locales/fr.json

```diff
@@ -217,77 +245,86 @@
-  "media-tile.joined-room": "Joint {relativeTime}",
+  "media-tile.joined-room": "A rejoint {relativeTime}",
   "mic-permissions-modal.grant-mic-permissions": "Accorder les autorisations au micro",
   "mic-permissions-modal.mic-access-needed": "Vous avez besoin d'un micro pour être entendu par les autres participants.{linebreak}Vous pourrez couper votre micro dans la salle.",
   "mic-permissions-modal.title": "Entrer par un appareil",
   "mic-setup-modal.check-mic": "Vérifiez votre micro et l'audio avant d'entrer.",
-  "mic-setup-modal.enter-room-button": "Entrer dans la Salle",
-  "mic-setup-modal.mic-disabled": "Micro Désactivé",
+  "mic-setup-modal.enter-room-button": "Entrer dans la salle",
+  "mic-setup-modal.mic-disabled": "Micro désactivé",
   "mic-setup-modal.mute-mic-toggle": "Couper mon micro",
-  "mic-setup-modal.test-audio": "Cliquer pour Tester l'Audio",
-  "mic-setup-modal.test-mic": "Parler pour Tester le micro",
+  "mic-setup-modal.test-audio": "Cliquer pour tester l'audio",
+  "mic-setup-modal.test-mic": "Parler pour tester le micro",
   "mic-setup-modal.title": "Réglages du micro",
   "more-menu-popover.title": "Plus",
-  "more-menu.close-room": "Fermer la Salle",
+  "more-menu.close-room": "Fermer la salle",
   "more-menu.community": "Communauté",
-  "more-menu.controls": "Controles",
-  "more-menu.create-room": "Créer une Salle",
-  "more-menu.enter-leave-room": "Quitter la Salle",
+  "more-menu.controls": "Commandes",
+  "more-menu.create-room": "Créer une salle",
+  "more-menu.enter-leave-room": "Quitter la salle",
   "more-menu.enter-streamer-mode": "Entrer en mode Streameur",
   "more-menu.exit-streamer-mode": "Sortir du mode Streameur",
-  "more-menu.favorite-room": "Salle Favorite",
-  "more-menu.favorite-rooms": "Salles Favorites",
+  "more-menu.favorite-room": "Salle favorite",
+  "more-menu.favorite-rooms": "Salles favorites",
   "more-menu.help": "Aide",
   "more-menu.invite": "Inviter",
+  "more-menu.preferences": "Préférences",
   "more-menu.privacy": "Notes sur la vie privée",
-  "more-menu.profile": "Changer de Nom & Avatar",
-  "more-menu.report-issue": "Signaler une Erreur",
+  "more-menu.profile": "Changer de nom & avatar",
+  "more-menu.report-issue": "Signaler une erreur",
   "more-menu.room": "Salle",
-  "more-menu.room-info": "Info de la Salle et Réglages",
+  "more-menu.room-info": "Info de la salle et réglages",
   "more-menu.sign-in": "Se connecter",
   "more-menu.sign-out": "Se déconnecter",
   "more-menu.start-tour": "Démarrer la visite guidée",
   "more-menu.support": "Support",
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
-  "oauth-screen.description": "Vous devrez vous connecter pour {provider} accéder à la salle.",
+  "more-menu.you": "Vous",
+  "more-menu.you-signed-in-as": "Vous (connecté(e) en tant que {email})",
```

Comment:
Why is there parenthesis ?

Reply:
English was "You (Signed in as: {email})"
What would you want instead, a comma? "Vous, connecté(e) en tant que {email}" ?

Reply:
That's already better, maybe with a verb: `Vous êtes connecté(e) en tant que {email}`

Reply:
While it's good translation if it was used as a sentence, here it's used mainly as a title for a group of links, see screenshot https://github.com/mozilla/hubs/pull/5003#issuecomment-1008111654
so I still prefer with the parentheses. What do others think?

Reply:
isn't it linked to Spoke? to be sure of the account used?

Reply:
Not really related to Spoke, but yes you can be sure with which email you are connected. Screenshot when connected:
![Capture d’écran de 2022-01-12 14-31-40](https://user-images.githubusercontent.com/112249/149150103-348783bf-a5d1-4e4a-b551-de07c7051538.png)

Reply:
Or maybe just: `Connecté(e) en tant que {email}` there is not so much space to print the full sentence.
Even if it's not `like` the english version we keep the sense and we there is not a lot of space.

Reply:
requesting a change here:

the string should be `"header.signed-in-as"` with it's default string being "Signed in as {email}"
For "You", I believe that is going to be `"people-sidebar.person-name.you"`

Reply:
See #5003 this part wasn't internationalized yet. All messages in this More menu, even the headers "Room" and "Support" are using the "more-menu." dotted name.
Separating the "You" and "Signed in as {email}" in different messages doesn't seem to be a good idea to me, because you will hard code the parentheses in the code? Some language may not really use parentheses for this. And be careful to right to left languages too. :)

Reply:
Thanks for the clarification, I didn't see the separate PR's but I will take a look. I think we should drop the parens and the word 'you' altogether if we are going to make changes at that level.